pkg/apis/uiplugin/v1alpha1/types.go

```diff
@@ -149,6 +149,8 @@ type LokiStackReference struct {
 
 // UIPluginSpec is the specification for desired state of UIPlugin.
 //
+// +kubebuilder:validation:XValidation:rule="self.type == 'TroubleshootingPanel' || !has(self.troubleshootingPanel)", message="TroubleshootingPanel type is required to use Troubleshooting Panel configuration"
```

Comment:
```suggestion
// +kubebuilder:validation:XValidation:rule="self.type == 'TroubleshootingPanel' || !has(self.troubleshootingPanel)", message="Troubleshooting panel configuration is only supported with the TroubleshootingPanel type"
```

Reply:
Updated the wording of the message of all the TroubleshootingPanel validations